test/test_device.py

```diff
@@ -24,7 +24,7 @@
 SUPPORTS_MULTISIG = {'ledger', 'trezor_1', 'digitalbitbox', 'keepkey', 'coldcard', 'trezor_t', 'jade'}
 SUPPORTS_EXTERNAL = {'ledger', 'trezor_1', 'digitalbitbox', 'keepkey', 'coldcard', 'trezor_t', 'jade'}
 SUPPORTS_OP_RETURN = {'ledger', 'digitalbitbox', 'trezor_1', 'trezor_t', 'keepkey', 'jade'}
-SUPPORTS_TAPROOT = {}
+SUPPORTS_TAPROOT = {"trezor_1", "trezor_t"}
```

Comment:
```suggestion
SUPPORTS_TAPROOT = {'trezor_1', 'trezor_t'}
```

Reply:
I'm trying to move the style to using double quotes rather than single.